assets/dev/js/frontend/utils/assets-loader.js

```diff
@@ -17,25 +17,21 @@ export default class AssetsLoader {
 	}
 
 	load( type, key ) {
-		return new Promise( ( resolve ) => {
-			const assetData = this.constructor.assets[ type ][ key ];
+		const assetData = this.constructor.assets[ type ][ key ];
```

Comment:
Use `AssetsLoader` instead of `this.constructor`

Reply:
Done.